main.nf

```diff
@@ -698,7 +698,7 @@ process sample_correlation {
 
     script: // This script is bundled with the pipeline, in NGI-RNAseq/bin/
     def rlocation = params.rlocation ?: ''
-    prefix = input_files[0].toString() - ~/(_R1)?(_trimmed)?(_val_1)?(\.fq)?(\.fastq)?(\.gz)?$/
+    prefix = input_files[0].toString() - 'Aligned.sortedByCoord.out_gene.featureCounts.txt' 
```

Comment:
Ok, let's merge and see what happens. Not the end of the world even if it doesn't work.